### PR TITLE
PP-5244 Sandbox Events Storage

### PR DESCRIPTION
**Pull request by RoryMalcolm**
- Create SandboxEvent class with mapper and dao
- Update Sandbox webhook resource to work with SandboxEvent
- Add test for SandboxEventDaoIT.

### CODE REVIEW

File: src/main/java/uk/gov/pay/directdebit/webhook/sandbox/resources/WebhookSandboxResource.java [hunk 1]
```diff
@@ -1,36 +1,60 @@
...
 
     @Inject
-    public WebhookSandboxResource(TransactionService transactionService) {
+    public WebhookSandboxResource(TransactionService transactionService, SandboxEventService sandboxEventService) {
         this.transactionService = transactionService;
+        this.sandboxEventService = sandboxEventService;
+    }
+    
+    enum SandboxEventAction {
+        SANDBOX_EVENT_ACTION
```
Comment (alexbishop1) [inline src/main/java/uk/gov/pay/directdebit/webhook/sandbox/resources/WebhookSandboxResource.java]: I think this (and the cause) should probably be `PAID_OUT` or something to reflect what’s already there

Comment (danworth) [reply src/main/java/uk/gov/pay/directdebit/webhook/sandbox/resources/WebhookSandboxResource.java]: Good idea I've updated.

**Comment by danworth** [comment]
> We might want to also think about having an integration test for the `WebhookSandboxResource`.

Shall we wait until we've expanded on what we want the Sandbox webhook to do before adding a ResourceIT (we could add as part of the events work we're planning once have a better idea)? I'd like to get this PR merged before end of sprint so we've got the Sandbox events ready for the next stories.

**Comment by alexbishop1** [comment]
Sure, we can wait until we have more significant functionality before adding an integration test.

**Pull request merged**